apps/condo/domains/user/schema/AuthenticateUserWithPhoneAndPasswordService.test.js

```diff
@@ -33,17 +31,33 @@ describe('Auth by phone and password', () => {
         const [, userAttrs] = await createTestUser(admin)
         const { phone, password } = userAttrs
         const client = await makeClient()
-        const res1 = await client.mutate(SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION, { phone, password: password + Math.random() })
-        expect(JSON.stringify(res1.errors)).toContain(WRONG_PASSWORD_ERROR)
+        const { errors } = await client.mutate(SIGNIN_BY_PHONE_AND_PASSWORD_MUTATION, { phone, password: password + Math.random() })
+        expect(errors).toMatchObject([{
+            message: 'Wrong password',
+            name: 'GraphQLError',
+            path: ['obj'],
+            extensions: {
+                mutation: 'authenticateUserWithPhoneAndPassword',
```

Comment:
you need to add `code` and `type`

Reply:
and probably translations ?

Reply:
Please, take a look at the `changePasswordWithToken` mutation in `ForgotPasswordActionService`.
Yet, this is only one mutation, that utilises all implemented features.
Next step will utilize the features in all another custom mutations, including this one.